internal/scaffold/ansible/go_mod.go

```diff
@@ -43,36 +43,41 @@ go 1.13
 
 require (
 	github.com/operator-framework/operator-sdk master
-	sigs.k8s.io/controller-runtime v0.4.0
+	github.com/spf13/pflag v1.0.5
+	k8s.io/api v0.17.4
+	k8s.io/apimachinery v0.17.4
+	k8s.io/client-go v12.0.0+incompatible
+	sigs.k8s.io/controller-runtime v0.5.2
 )
 
-// Pinned to kubernetes-1.16.2
+// Pinned to kubernetes-1.17.4
```

Comment:
Ditto, we probably don't need most of these anymore.

Reply:
We may be able to remove, however, I am facing issues locally when I try to scaffold a new project with the local source code whiteout the pinned versions. 

However, see that it will look to master `github.com/operator-framework/operator-sdk master`

So, I understand that we will just able to change it if possible in a follow-up PR.